modules/ROOT/pages/add-api-to-unified-schema.adoc

```diff
@@ -50,6 +49,7 @@ The first task to adding an API to the unified schema is adding the API to Anypo
 +
 * *No Auth*: Use if your API is public.
 * *Basic Auth*: Use if you authenticate to your API using a username and password.
+* *Pass-through*: Use if you authenticate to your API by passing authorization headers.
```

Comment:
@mauro-dentoni this is for adding pass-through auth when you add an api to the unified schema

Reply:
Looks fine